### DISCUSSION TITLE
Channels with call signs in the tvg-id

### DISCUSSION THREAD

**Discussion by jkemp814** [iptv-org/community#225] [Show and tell]
I have created a gist: https://gist.github.com/jkemp814/41151a0f7c45d5961563c7531c664e54

That shows channels with call signs in the tvg-id tag. 

Can someone take a look and see what we should do to handle this? 

Should we leave it as it is?

Some of these channels do not show up around other news  channels because they start with 'K' or 'W', so they are way down the list.

How should these mainly news and public broadcasting channels be formatted? 

I need an example, please. ABC 15 Phoenix (KNXV) - ABC 7 Detroit (WXYZ)?

Any advice would be greatly appreciated.

**Comment by jkemp814** [comment]
Using call sign list

**Comment by Sphinxroot** [comment]
the main problem is to have the proper EPG (tvg-url ) that have those  tvg-id

there so many source   for them   and many use different id  

exemple 
https://github.com/epgshare01/share01
https://github.com/iptv-org/epg/blob/master/sites/tvtv.us.channels.xml
https://github.com/SilentButeo2/webgrabplus-siteinipack/tree/master/siteini.pack

**Reply by jkemp814** [reply]
I have notice that. I have been using the list from: 
https://github.com/iptv-org/epg/blob/master/codes.csv

Also:
https://en.wikipedia.org/wiki/List_of_television_stations_in_the_United_States_by_call_sign_(initial_letter_K)
https://en.wikipedia.org/wiki/List_of_television_stations_in_the_United_States_by_call_sign_(initial_letter_W)
https://en.wikipedia.org/wiki/List_of_television_stations_in_Mexico

Other countries are linked on that same page.
